packages/css/src/components/icon/icon.scss

```diff
@@ -4,10 +4,12 @@
  */
```

Comment:
Maybe out of scope, but the Icon Button currently isn't square. It's 40x44px on wide screens.

Reply:
> Maybe out of scope, but the Icon Button currently isn't square. It's 40x44px on wide screens.

I’ve made an Icon Button `display: inline-flex` to remove the phantom white space that `inline-block` introduces. Our Button already did the same.

> And in Alert and Dialog it's wider than it's tall, for some reason?

Not sure why that is, I’ll make a note for it.